app/policies/user_mentee_application_policy.rb

```diff
@@ -19,6 +19,16 @@ def update?
     matching_user?
   end
 
+  class Scope < Scope
+    def resolve
+      if user.admin?
+        scope.all
+      else
+        scope.where(user_id: user.id)
```

Comment:
I don't think we need this because the index should only be applicable to admin. For the show page, it will just be a find + show pundit check. (we also don't want to be showing the applicant the notes we leave 😓 )